server/app/services/program/ProgramType.java

```diff
@@ -21,4 +21,11 @@ public enum ProgramType {
   public String getValue() {
     return dbValue;
   }
+
+  public static ProgramType fromValue(String value) {
+    if (value.equals("common_intake_form")) {
+      return COMMON_INTAKE_FORM;
+    }
+    return DEFAULT;
+  }
```

Comment:
```suggestion
  public static ProgramType fromValue(String value) {
    for (ProgramType programType : values()) {
        if (programType.value.equals(value)) {
            return programType;
        }
    }
    throw new IllegalArgumentException("Unknown value: " + value);
  }
```

(question) Do you expect `value` to ever be an empty string or a string value not part of the enum? If no, just loop through and throw as shown in the above sample. If yes, in what case(s) would this default  to `DEFAULT`?